example/gardener-local/gardenlet/values.yaml

```diff
@@ -36,6 +36,7 @@ config:
     CopyEtcdBackupsDuringControlPlaneMigration: true
     DefaultSeccompProfile: true
     CoreDNSQueryRewriting: true
+    IPv6SingleStack: true
```

Comment:
Same here - doesn't this mean IPv6 single stack will be enabled in all local Gardener environments?

Reply:
yes, but this just means you can provide IPv6 addresses and Gardener validates them. IPv4 does also work.

Reply:
https://github.com/gardener/gardener/pull/7561#discussion_r1119256147 is related. Maybe yes, maybe the feature gate will lead to behavioural changes in the future. Perhaps it's safer to disable it by default even in the local environments.